spec/paths/v1@token_info@.yaml

```diff
@@ -0,0 +1,16 @@
+get:
+  operationId: getTokenInfo
+  summary: Get your token information
+  description: Gives token information
+  produces:
+    - application/json
+  parameters: []
+  responses:
+    '200':
+      description: successful operation
+      schema:
+        $ref: '#/definitions/TokenInfoV1'
+  security:
+    - apiKey: []
+  tags:
+    - TokenInfo
```

Comment:
Tags will appear on left column in API documentation. So this casing won't look good there. These are more like groups. Just make it `Token`. I think you'll have this in `x-tag-groups` in `openapi.yaml`.